tests/miscTests/ValidateLocalInfoXML.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * 
+ */
+function libxml_display_error($error)
+{
+    $return = "<br/>\n";
+
+    switch ($error->level) {
+    case LIBXML_ERR_WARNING:
+        $return .= "<b>Warning $error->code</b>: ";
+        break;
+    case LIBXML_ERR_ERROR:$return .= "<b>Error $error->code</b>: ";
+        break;
+    case LIBXML_ERR_FATAL:$return .= "<b>Fatal Error $error->code</b>: ";
+        break;
+    }
+    $return .= trim($error->message);
+
+    if ($error->file) {
+        $return .= " in <b>$error->file</b>";
+    }
+    $return .= " on line <b>$error->line</b>\n";
+    return $return;
+}
+/**
+ * 
+ */
+function libxml_display_errors() 
```

Comment:
This function seems similar enough to `libxml_display_errors` in `htdocs/web_portal/GOCDB_monitor/validate_local_info_xml.php`, perhaps it could be reworked so this function isnt needed and instead we use a `require` statement?